modules/era/sql/weapon_skills.sql

```diff
@@ -0,0 +1,271 @@
+    (216,'coronach',0x00000000000000000000000000000000000000000000,26,0,0,226,2000,21,1,14,12,0,1,0),
+    (217,'trueflight',0x00000000000000000000010000000000000000000000,26,0,128,227,2000,21,1,12,4,0,1,33),
+    (218,'leaden_salute',0x00000000000000000000000000000000010000000000,26,0,255,228,2000,21,1,9,1,0,1,34);
+
+    -- OOE
+DELETE FROM `weapon_skills` WHERE
+    -- Hand to Hand
+    name = "tornado_kick"       or -- June 2010
+    name = "victory_smite"      or -- September 2010
+    name = "shijin_spiral"      or -- December 2011
+    name = "final_paradise"     or -- May 2018
+
+    -- Daggers
+    name = "aeolian_edge"       or -- June 2010
+    name = "rudras_storm"       or -- September 2010
+
+    -- Swords
+    name = "sanguine_blade"     or -- June 2010
+    name = "chant_du_cygne"     or -- September 2010
+    name = "requiescat"         or -- December 2011
+    name = "knights_of_rotund"  or -- July 2017
+
+    -- Great Swords
+    name = "herculean_slash"    or -- June 2010
+    name = "torcleaver"         or -- September 2010
+    name = "resolution"         or -- December 2011
+    name = "dimidiation"        or -- July 2014
+
+    -- Axes
+    name = "bora_axe"           or -- June 2010
+    name = "cloudsplitter"      or -- September 2010
+    name = "ruinator"           or -- December 2011
+
+    -- Great Axes
+    name = "fell_cleave"        or -- June 2010
+    name = "ukkos_fury"         or -- September 2010
+    name = "upheaval"           or -- December 2011
+
+    -- Scythe
+    name = "infernal_scythe"    or -- June 2010
+    name = "quietus"            or -- September 2010
+    name = "entropy"            or -- December 2011
+
+    -- Polearm
+    name = "sonic_thrust"       or -- June 2010
+    name = "camlanns_torment"   or -- September 2010
+    name = "stardiver"          or -- December 2011
+
+    -- Katana
+    name = "blade_yu"           or -- June 2010
+    name = "blade_hi"           or -- September 2010
+    name = "blade_shun"         or -- December 2011
+
+    -- Great Katana
+    name = "tachi_ageha"        or -- June 2010
+    name = "tachi_fudo"         or -- September 2010
+    name = "tachi_shoha"        or -- December 2011
+    name = "tachi_suikawari"    or -- August 2018
+
+    -- Clubs
+    name = "flash_nova"         or -- June 2010
+    name = "dagan"              or -- Sepember 2010
+    name = "realmrazer"         or -- December 2011
+
+    -- Staffs
+    name = "cataclysm"          or -- June 2010
+    name = "myrkr"              or -- Sepember 2010
+    name = "shattersoul"        or -- December 2011
+
+    -- Archery
+    name = "refulgent_arrow"    or -- June 2010
+    name = "jishnus_radiance"   or -- September 2010
+    name = "apex_arrow"         or -- December 2011
+
+    -- Marksmanship
+    name = "numbing_shot"       or -- June 2010
+    name = "wildfire"           or -- September 2010
+    name = "last_stand";           -- December 2011
+    
```

Comment:
Newline at end, and have we tested ANY of this yet?

Reply:
Fixed